examples/index.json

```diff
@@ -957,14 +957,36 @@
         "title": "RabbitMQ",
         "column": 3,
         "category": "Working over the network",
-        "samples": [{
-            "name": "Basic Producer",
-            "url": "rabbitmq-simple-producer"
-        },
-          {
-            "name": "Basic Consumer",
-            "url": "rabbitmq-simple-consumer"
-          }]
+        "samples": [
+            {
+                "name": "RabbitMQ Producer",
```

Comment:
Shall we remove the RabbitMQ part, since this is already under RabbitMQ section?

Reply:
Resolved